lightspeed/modules/proc_login-vscode-extension.adoc

```diff
@@ -0,0 +1,26 @@
+:_content-type: PROCEDURE
+
+[id="login-vscode-extension_{context}"]
+= Logging in to Ansible Lightspeed through the Ansible VS Code extension
+
+After installing and configuring the VS Code extension, you can log in to the Ansible Lightspeed service.
+
+.Procedure
+
+. Open the VS Code application.
+. Use one of the following ways to connect to the Ansible Lightspeed service. 
+** Using the *Connect* button:
+.. From the navigation menu, click the *Ansible* icon. 
+.. Under Ansible Lightspeed Login, click *Connect*.
+** Using the *Accounts* button:
+.. From the navigation menu, click the *Accounts icon > Sign in with Ansible Lightspeed*.
```

Comment:
Is the menu item "Accounts" or does it include the word "icon"? Just a suggestion. Or will an image of the icon be used at some point?

```suggestion
.. From the navigation menu, click *Accounts* icon > Sign in with Ansible Lightspeed*.
```

Reply:
It's an icon titled Accounts. I made the change you recommended, but I'm not adding an image since things are changing dynamically and maintenance gets difficult. Thank you.

Reply:
LGTM!